site-assets/learn.js

```diff
@@ -0,0 +1,215 @@
+/* global _ */
+(function ($) {
+    'use strict';
+
+    /* jshint ignore:start */
+    // Underscore's Template Module
+    // Courtesy of underscorejs.org
+    var _ = (function (_) {
```

Comment:
Agree with that, I just followed what's done in https://github.com/tastejs/todomvc-common/blob/master/base.js
I am assuming it was done to avoid loading underscore?